Tools/Scripts/filter-build-webkit

```diff
@@ -395,6 +405,11 @@ sub shouldIgnoreLine($$)
     return 1 if $line =~ /^.* Run script build phase '/;
     return 1 if $line =~ /^.* Traditional headermap style is no longer supported;/;
     return 1 if $line =~ /^.* Skipping duplicate build file in/;
+    # This is most likely due Xcode and the command-line compilation race to open the same .dia files.
+    return 1 if $line =~ /warning: Could not read serialized diagnostics file: error\(\"Failed to open diagnostics file\"\)/;
+    # https://bugs.webkit.org/show_bug.cgi?id=175997
```

Comment:
I'd expand this as "The next line should be removed once https://webkit.org/b/175997 is fixed."